odl_multigrid/graphics.py

```diff
@@ -102,6 +102,48 @@ def show(data, eps=0.0):
     # TODO: set aspect from physical sizes
     ax.set_aspect('auto', 'box')
 
+def show_all(data, low = None, high = None):
+    """Show all reconstructed resolutions in a single image. For multiple ROIs"""
+    # Lazy import, can be slow
+    import matplotlib as mpl
+    import matplotlib.pyplot as plt
+    from matplotlib.image import BboxImage
+    from matplotlib.transforms import Bbox, TransformedBbox
+
+    fig, ax = plt.subplots()
+    
```

Comment:
Remove extra line

Reply:
Modified in a2a42d3

Reply:
...I think? :confused: Or did I miss which line you were after?